policyengine_us/variables/gov/states/nh/tax/income/exemptions/nh_base_exemption.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class nh_base_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Hampshire base exemption household level"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NH
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
```

Comment:
Are you sure this is per person rather than per filer? It says each `taxpayer`.

Also we conventionally compare exemptions to gross income in the taxable income computation, not in the exemption formula. So this can just be `adds = "gov.states.nh.tax.income.exemptions.amount.base"` rather than having a `formula`.